networking/hardware_networks/about-sriov.adoc

```diff
@@ -17,6 +17,14 @@ The SR-IOV network device driver for the device determines how the VF is exposed
 
 You can use SR-IOV network devices with additional networks on your {product-title} cluster installed on bare metal or {rh-openstack-first} infrastructure for applications that require high bandwidth or low latency.
 
+You can configure multi-network policies for SR-IOV networks. The support for this is technology preview and SR-IOV additional networks are only supported with kernel NICs. They are not supported for DPDK applications.
+
+[NOTE]
+====
+Creating multi-network policies on SR-IOV networks may not deliver the same performance to applications compared to SR-IOV networks without a multi-network policy configured. 
+====
+
```

Comment:
consider removing to avoid potential whitespace/merge conflicts in the future if it does not affect your formatting.